web_transport/sdk/impl/web_transport_server_session.cc

```diff
@@ -119,6 +125,8 @@ void WebTransportServerSession::SetVisitor(
 }
 
 const ConnectionStats& WebTransportServerSession::GetStats() {
+  const auto& stats = http3_session_->connection()->GetStats();
+  stats_.estimated_bandwidth = stats.estimated_bandwidth.ToBitsPerSecond();
```

Comment:
Now that the stats contains quite many info instead of only estimated bandwidth, I suggest you expose standard WebTransport stats instead of available bandwidth (not useful in most case because your read/write still depends on ReadableBytes() WirtableBytes() on the stream.

Reply:
[WebTransportStats](https://w3c.github.io/webtransport/#dictdef-webtransportstats) is going to be revised. We can add more when it is finalized.

Reply:
Can you provide the link to the PR that revises the spec for WebTransportStats? Meanwhile I will +1 for this PR.

Reply:
I think no pull request is ready at this time, but there is a notification says it'll be revised.